api/inventory/v1/agents.proto

```diff
@@ -666,25 +660,14 @@ message GetAgentLogsResponse {
   uint32 agent_config_log_lines_count = 2;
 }
 
-// Add PMMAgent
-
-message AddPMMAgentParams {
-  // Node identifier where this instance runs.
-  string runs_on_node_id = 1 [(validate.rules).string.min_len = 1];
-  // Custom user-assigned labels.
-  map<string, string> custom_labels = 2;
-}
-
 // TODO Change PMMAgent?
 
 // TODO Add VMAgent?
 
 // TODO Change VMAgent?
```

Comment:
let's drop it

Reply:
Why drop? Do we not want to implement it?

Reply:
Probably no

Reply:
Done.